src/fetching/tasks/patchTaskItem.ts

```diff
@@ -0,0 +1,23 @@
+import { defaultFetch } from '..';
+import { ZetkinSmartSearchFilter, ZetkinTask } from '../../types/zetkin';
+
+const patchTaskItem = (orgId : string, taskId: string, fetch = defaultFetch) => {
```

Comment:
I've added this to the august refactor issue, but why are we typing the params in the request functions as strings? They get converted to strings automatically in the template literal, and it would reduce verbosity when we don't have to pass them as strings on the pages.

Reply:
I think because they usually come into the program (from the URL) as strings. But when they do I think they're actually untyped, so they need to be type cast anyway.

But of course there are other sources for these IDs as well, where they're typed differently. Basically I don't think there's any silver bullet for these.

Reply:
we can do `string` or `number` but usually they come from the router query which I think is going to be strings.